driver-core/src/test/java/com/datastax/driver/core/PreparedStatementTest.java

```diff
@@ -334,13 +338,30 @@ public void prepareStatementInheritPropertiesTest() {
 
         RegularStatement toPrepare = new SimpleStatement("SELECT * FROM test WHERE k=?");
         toPrepare.setConsistencyLevel(ConsistencyLevel.QUORUM);
+        toPrepare.setSerialConsistencyLevel(ConsistencyLevel.LOCAL_SERIAL);
```

Comment:
Nice, I like that you tested that all the other properties get propagated as well 👍